src/ex2/PokemonClient.js

```diff
@@ -0,0 +1,30 @@
+class PokemonClinet {
+  constructor() {
+
+  }
+  fetchPokemon(arr) {
+    const responses = []; 
+    arr.forEach((id) => {
+      const data = fetch("https:pokeapi.co/api/v2/pokemon/" + id);
+      responses.push(data);
+    });
+       return Promise.all(responses).then((res) =>Promise.all (res.map(r=>r.json())));
+  }
+  
+  async checkByPokemonName(name) {
+    const response = await fetch(
+      `https:pokeapi.co/api/v2/pokemon?limit=100000&offset=0`
+    );
+    const pokemonsArrList = await response.json();
+      let res= null
+    for (const obj of pokemonsArrList.results) {
```

Comment:
use meaningful names
```suggestion
    for (const pokemon of pokemonsArrList.results) {
```